### PR TITLE
Improve Sass error output

### LINKED ISSUE #293: More descriptive errors for undefined variables
Hi, how can I get the error log to be more descriptive to include things like file location and line number?

For example, currently the errors look like:
```
Running "sass:dist" (sass) task
Fatal error: Undefined variable: "$link-color".
```

However, ideally I'd like something like:

```
Running "sass:dist" (sass) task
>> Error: Undefined variable: "$link-color".
>>         on line 145 of ../library/css/scss/imports/abstracts/_variables.scss
>>         from line 12 of ../library/css/scss/imports/_abstracts.scss
>>         from line 9 of ../library/css/scss/admin/login.scss
>> >> $link-hover: darken($link-color, 9%);
>> 
>>    --------------------^
```

### PR DESCRIPTION
**Pull request by toptalo**
Fixes #293

### CODE REVIEW

File: tasks/sass.js [hunk 1]
```diff
@@ -36,6 +36,8 @@ module.exports = grunt => {
 					grunt.file.write(filePath, result.map);
 				}
 			}));
-		})().catch(grunt.fatal).then(done);
+		})().catch(error => {
+			grunt.fatal(error.formatted);
```
Comment (sindresorhus) [inline tasks/sass.js]: The error here is not guaranteed to only be the Sass error, so the `.formatted` property may or may not exist.

Comment (toptalo) [reply tasks/sass.js]: If I change to `error.formatted || error` ?

Comment (toptalo) [reply tasks/sass.js]: Or, maybe, wrap `render` with `try ... catch` and throw error with formatted message?

Comment (sindresorhus) [reply tasks/sass.js]: > If I change to error.formatted || error ?

👍

**Pull request merged**